### PR TITLE
Add Kong service mesh

### PR DESCRIPTION
**Pull request by coopr**
Kong has announced a service mesh configuration option https://konghq.com/solutions/service-mesh/ and will soon ship a Kong 1.0 release candidate including this functionality.

### CODE REVIEW

**Comment by coopr** [comment]
Hi @leecalcote can I do anything to help get this merged and deployed?

File: _data/cat/meshes.yml [hunk 1]
```diff
@@ -23,14 +23,23 @@
   multi-tenant: "Experimental"
   desc: "An open platform to connect, monitor, and secure microservices. Governed by Google and IBM."
 
+- name: Kong
+  link: https://konghq.com/solutions/service-mesh/
```
Comment (leecalcote) [inline _data/cat/meshes.yml]: Hi @coopr, can we identify a suitable description for Kong Mesh?

`desc:` "Kong - a microservices API gateway."` - - would work for Kong itself. How can we better describe Kong Mesh?

Outside of this description, I'm looking to qualify Kong Mesh as a service mesh. Does Kong Mesh have its own control plane? Does it integration with other meshes, like Istio, as a proxy to displace Envoy?

Comment (coopr) [reply _data/cat/meshes.yml]: > Kong, the world's most popular OSS API Gateway, now slices, dices, and service meshes!

> Same Kong, new way to use it.

> Clean up your service mess with the Kong service mesh

I'll stop now 😄 

Will update PR now...

File: _data/cat/meshes.yml [hunk 2]
```diff
@@ -23,14 +23,24 @@
...
+
 - name: Linkerd 1.x
   desc: "Linkerd - hosted by the CNCF and built on top of Twitter Finagle. Linkerd includes both a proxying data plane and the Namerd (“namer-dee”) control plane all in one package. "
   link: https://linkerd.io
   h2: "Yes"
   tcp-web: "Yes"
 
 - name: Linkerd 2.x (Conduit)
-  desc: "Conduit - A Kubernetes-native (only) service mesh announced as a project in December 2017. In contrast to Istio and in learning from Linkerd, Conduit’s design principles revolve around a minimalist architecture and zero config philosophy, optimizing for streamlined setup. 
+  desc: "Conduit - A Kubernetes-native (only) service mesh announced as a project in December 2017. In contrast to Istio and in learning from Linkerd, Conduit’s design principles revolve around a minimalist architecture and zero config philosophy, optimizing for streamlined setup.
```
Comment (coopr) [inline _data/cat/meshes.yml]: Sorry, my text editor did that without me noticing.

**Comment by leecalcote** [comment]
Thank you, @coopr!

**Pull request merged**